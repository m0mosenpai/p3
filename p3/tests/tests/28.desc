diff will return with exit value 1.
