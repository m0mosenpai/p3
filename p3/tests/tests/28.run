../solution/wsh tests/28.wsh 2>/dev/null
