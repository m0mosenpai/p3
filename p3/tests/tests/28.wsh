diff tests/27.run tests/28.run
