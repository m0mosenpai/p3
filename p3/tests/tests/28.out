1c1
< timeout 5m valgrind --leak-check=full --show-leak-kinds=all --track-origins=yes --verbose ../solution/wsh <tests/27.wsh 2>&1 >/dev/null | grep "no leaks are possible" > /dev/null
---
> ../solution/wsh tests/28.wsh 2>/dev/null
